content/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services.md

```diff
@@ -10,59 +10,65 @@ toc: true
 weight: 400
 ---
 
-This guide explains how to use the NGINX Open Source or F5 NGINX Plus Ingress Controller for Kubernetes as the Ingress controller for a Kubernetes cluster hosted in the Amazon Elastic Kubernetes Service (EKS).
+This guide explains how to use NGINX Open Source or F5 NGINX Plus Ingress Controller for Amazon Elastic Kubernetes Services (EKS).
 
-**Note:** The instructions in this document apply to both the NGINX and NGINX Plus Ingress Controllers for Kubernetes. For ease of reading, the document refers to NGINX Plus only.
+**Note:** These instructions apply to both the NGINX and NGINX Plus Ingress Controllers for Kubernetes. For ease of reading, the document refers to NGINX Plus only.
 
 
 <span id="prereqs"></span>
 ## Prerequisites
 
 - [An AWS account](https://docs.aws.amazon.com/AmazonSimpleDB/latest/DeveloperGuide/AboutAWSAccounts.html).
-- A prebuilt image of the NGINX or NGINX Plus Ingress Controller for Kubernetes. For NGINX Open Source, NGINX provides a prebuilt image on DockerHub, or you can build your own with our instructions. For NGINX Plus, you must build an image.
-
-  **Note:** If you build the image, do not push it to a public registry.  Run the following <span style="white-space: nowrap;">`make` `container`</span> command. Include the `PREFIX` argument to specify the repo in your private registry where the container is created; this also sets the name that you can later use to reference the image, instead of its numerical ID. In this example we set it to <span style="white-space: nowrap; font-weight:bold;">nginx/nginx-plus-ingress</span>.
+- A prebuilt image of the NGINX or NGINX Plus Ingress Controller for Kubernetes. 
+  - For NGINX Open Source you can use the pre-built image [on DockerHub](https://hub.docker.com/r/nginx/nginx-ingress/). You can also build your own image.
+  - For NGINX Plus, you must build an image.
 
+**Note:** If you build the image, do not push it to a public registry.  Run the <span style="white-space: nowrap;">`make` `container`</span> command below.
 
 ```shell
 make container DOCKERFILE=DockerfileForPlus PREFIX=nginx/nginx-plus-ingress
 ```
 
+The `PREFIX` argument specifies the repo name in your private container registry. In this example, we set it to <span style="white-space: nowrap; font-weight:bold;">nginx/nginx-plus-ingress</span>. You can later use that name to reference the image instead of its numerical ID.
+   
```

Comment:
```suggestion
The `PREFIX` argument specifies the repo name in your private container registry. In this example, we set it to `nginx/nginx-plus-ingress`. You can later use that name to reference the image instead of its numerical ID.
   
```